MC/config/PWGHF/pythia8/custom/pythia_onia_X.cfg

```diff
@@ -0,0 +1,64 @@
+### beams
+Beams:idA 2212                  # proton
+Beams:idB 2212                  # proton
+Beams:eCM 14000.                # GeV
+#663:addChannel = 1 0. 0 1 -1
+#9920443:addParticle = X(3872) 3 0 0 3.87196 0.00012
```

Comment:
We also need to make sure that Geant knows this particle. To be considered.

Reply:
Just a test

Reply:
GEANT will not see the X(3872) because it decays at the generator level

Reply:
this file should also be adjusted.
the decay part must be decoupled from the process selection part.

The process selection part must be somehow encapsulated in the `mkpy8cfg.py` script.
Whereas the decay part should go into a decay cfg file.

Reply:
Sounds good. For the Geant discussion, this is correct for the X. But I think there could be particles that we add that we might want to propagate (?). Thanks for the reference to mkpy8cfg.py. I will have a look.